stabilizer/src/main/java/com/hazelcast/stabilizer/test/utils/TestUtils.java

```diff
@@ -114,6 +115,12 @@ public static void assertTrueEventually(AssertTask task) {
         assertTrueEventually(task, ASSERT_TRUE_EVENTUALLY_TIMEOUT);
     }
 
+    public static void assertEqualsByteArray(byte[] expected,byte[] current){
+        for(int k = 0; k < expected.length; k++){
```

Comment:
What if the arrays have difference sizes. EIther you get an index out of bounds or you are not checking every element.